bundle/permissions/permission_report.go

```diff
@@ -0,0 +1,50 @@
+package permissions
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/log"
+)
+
+// ReportPossiblePermissionDenied generates a diagnostic message when a permission denied error is encountered.
+//
+// Note that since the workspace API doesn't always distinguish between permission denied and path errors,
+// we must treat this as a "possible permission error". See acquire.go for more about this.
+func ReportPossiblePermissionDenied(ctx context.Context, b *bundle.Bundle, path string) diag.Diagnostics {
+	log.Errorf(ctx, "Failed to update, encountered possible permission error: %v", path)
+
+	user := b.Config.Workspace.CurrentUser.DisplayName
+	if user == "" {
+		user = b.Config.Workspace.CurrentUser.UserName
+	}
```

Comment:
Because of the other PR today, I now look at this and see it will show the display name for regular users.

It will affect the error message shown. I think for regular users we better show the email address.

Reply:
I changed it to just show the email address in errors.